mycroft/skills/media/vocab/en-us/FromKeyword.voc

```diff
@@ -0,0 +1 @@
+from
```

Comment:
I used that in the beginning but I went over to the from keyword to make it easier to translate the skill... I do a dirty hack in the spotify skill to allow some free searches...

Any way I've removed it from the media common vocabulary and just use it in the spotify skill. I'll see if I can figure out how to remove it from there as well...